File: MonoArt/Packages/AsciiCameraKit/.build/arm64-apple-macosx/debug/PocketSVG.build/DerivedSources/resource_bundle_accessor.m
#import <Foundation/Foundation.h>

NSBundle* PocketSVG_SWIFTPM_MODULE_BUNDLE() {
    NSURL *bundleURL = [[[NSBundle mainBundle] bundleURL] URLByAppendingPathComponent:@"PocketSVG_PocketSVG.bundle"];

    NSBundle *preferredBundle = [NSBundle bundleWithURL:bundleURL];
    if (preferredBundle == nil) {
      return [NSBundle bundleWithPath:@"/Users/barryalien/Documents/code/MonoArt/MonoArt/Packages/AsciiCameraKit/.build/arm64-apple-macosx/debug/PocketSVG_PocketSVG.bundle"];
    }

    return preferredBundle;
}

File: MonoArt/Packages/AsciiCameraKit/.build/arm64-apple-macosx/debug/AsciiUI.build/DerivedSources/resource_bundle_accessor.swift
import Foundation

extension Foundation.Bundle {
    static let module: Bundle = {
        let mainPath = Bundle.main.bundleURL.appendingPathComponent("AsciiCameraKit_AsciiUI.bundle").path
        let buildPath = "/Users/barryalien/Documents/code/MonoArt/MonoArt/Packages/AsciiCameraKit/.build/arm64-apple-macosx/debug/AsciiCameraKit_AsciiUI.bundle"

        let preferredBundle = Bundle(path: mainPath)

        guard let bundle = preferredBundle ?? Bundle(path: buildPath) else {
            // Users can write a function called fatalError themselves, we should be resilient against that.
            Swift.fatalError("could not load resource bundle: from \(mainPath) or \(buildPath)")
        }

        return bundle
    }()
}